src/native/libs/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.20)
+
```

Comment:
```suggestion
```
We need this only for the libraries that we support static linking for. The fewer places with cmake version, the better.